netcam_rtsp.c

```diff
@@ -258,15 +258,31 @@ struct rtsp_context *rtsp_new_context(void){
 static int netcam_interrupt_rtsp(void *ctx){
     struct rtsp_context *rtsp = (struct rtsp_context *)ctx;
 
-    if (rtsp->readingframe != 1) {
+    if (rtsp->status == RTSP_CONNECTED) {
```

Comment:
The main reason was not wanting to wrap every single call to a ffmpeg function and instead leave calls outside of the open context / reading frame to a default.  It would get pretty messy dealing with all the calls separately.